<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lista de Carros</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPushButton" name="btnProcurar">
     <property name="text">
      <string>Procurar</string>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/img/busca.png</normaloff>:/img/busca.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="txtBuscaPlaca">
     <property name="inputMask">
      <string>AAA-9999</string>
     </property>
     <property name="placeholderText">
      <string>Placa...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="btnSaida">
     <property name="text">
      <string>Registrar Saída</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeWidget" name="twListaCarros">
     <column>
      <property name="text">
       <string>Placa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modelo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Entrada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Saida</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="btnListarCarros">
     <property name="text">
      <string>Listar carros estacionados</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblTelaCarros">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Carros Estacionados:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="btnCadastrar">
     <property name="text">
      <string>Cadastrar Veículo</string>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/img/1_1-128.png</normaloff>:/img/1_1-128.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
